noxfile.py

```diff
@@ -0,0 +1,70 @@
+import nox
+import os
+import sys
+
+
+@nox.session(name="pybamm-requires")
+def run_pybamm_requires(session):
+    if sys.platform != "win32" or sys.platform != "darwin":
+        session.install("wget", "cmake")
+        session.run("python", "scripts/install_KLU_Sundials.py")
+        session.run(
+            "git",
+            "clone",
+            "https://github.com/pybind/pybind11.git",
+            "pybind11/",
+            external=True,
+        )
+
+
+@nox.session(name="coverage")
+def run_coverage(session):
+    homedir = os.getenv("HOME")
+    session.env["SUNDIALS_INST"] = session.env.get("SUNDIALS_INST", f"{homedir}/.local")
+    session.env[
+        "LD_LIBRARY_PATH"
+    ] = f"{homedir}/.local/lib:{session.env.get('LD_LIBRARY_PATH')}"
+    session.install("coverage", "scikits.odes")
+    session.run("pip", "install", "-e", ".[dev]")
+    if sys.platform != "win32" or sys.platform != "darwin":
+        session.run("pybamm_install_jax")
+    session.run("coverage", "run", "run-tests.py", "--nosub")
```

Comment:
```suggestion
    session.run("coverage", "run", "--rcfile=.coveragerc", "run-tests.py", "--nosub")
```

Just to ensure that it reads from the file, not needed though

Reply:
Also, don't mind me barging in with a review lol—I suggested these changes because I received a review request notification on mobile, but I don't see it on web

Reply:
It's totally cool, you can always suggest changes if you feel like, i'm glad you took a look into it.